### PR TITLE
add public code tag line

### PR DESCRIPTION
**Pull request by rrhyne**
Adds a tag line below the logo to the Cloud application.

image

### CODE REVIEW

File: web/src/search/input/SearchPage.tsx [hunk 1]
```diff
@@ -87,6 +87,7 @@ export const SearchPage: React.FunctionComponent<Props> = props => {
     return (
         <div className="search-page">
             <BrandLogo className="search-page__logo" isLightTheme={props.isLightTheme} />
+            {props.isSourcegraphDotCom ? <div className="search-page__cloud-tag-line">Search public code</div> : null}
```
Comment (eseliger) [inline web/src/search/input/SearchPage.tsx:90]:
```suggestion
            {props.isSourcegraphDotCom && <div className="search-page__cloud-tag-line">Search public code</div>}
```

is slightly easier to read I think :)

**Comment by rrhyne** [comment]
@eseliger screenshot added to PR.

**Comment by felixfbecker** [comment]
Should this maybe be the input placeholder instead? In interactive mode, we also have a placeholder that says "Search code...", the tagline feels a bit redundant to that. The fact that plain text mode does not have a placeholder is an inconsistency that we should solve regardless I think.

**Comment by rrhyne** [comment]
> Should this maybe be the input placeholder instead? In interactive mode, we also have a placeholder that says "Search code...", the tagline feels a bit redundant to that. The fact that plain text mode does not have a placeholder is an inconsistency that we should solve regardless I think.

I considered this, but no, a placeholder will not solve the issue. Marketing would like to separate the enterprise product from the cloud product so that expectations are set appropriately. 

I agree with the placeholder inconsistency you mention and do feel that will be redundant. Perhaps now the answer is to remove the placeholder for cloud environments?

File: web/src/search/input/SearchPage.tsx (hunk 1 above)
Comment (rrhyne) [reply web/src/search/input/SearchPage.tsx]: Should be fixed.

**Comment by felixfbecker** [comment]
> I agree with the placeholder inconsistency you mention and do feel that will be redundant. Perhaps now the answer is to remove the placeholder for cloud environments?

Sounds good to me and should be straight-forward to change

**Pull request merged**